packages/manager/modules/product-offers/src/index.js

```diff
@@ -0,0 +1,36 @@
+import angular from 'angular';
+import 'angular-translate';
+import 'ovh-ui-angular';
+import ngOvhContracts from '@ovh-ux/ng-ovh-contracts';
+import ngOvhPaymentMethod from '@ovh-ux/ng-ovh-payment-method';
+import ngOvhWebUniverseComponents from '@ovh-ux/ng-ovh-web-universe-components';
+import ngTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
+
+import detachService from './services/product-offers-detach.service';
+import service from './services/product-offers.service';
+
+import component from './product-offers.component';
+import pricingConstants from './pricing/pricing.constants';
+import workflowConstants from './workflows/product-offers-workflow.constants';
+import extraPricingDetail from './pricing/extra-pricing-detail';
+
+const moduleName = 'ovhManagerProductOffers';
+
+angular
+  .module(moduleName, [
+    'oui',
+    extraPricingDetail,
+    ngOvhContracts,
+    ngOvhPaymentMethod,
+    ngOvhWebUniverseComponents,
+    ngTranslateAsyncLoader,
+    'pascalprecht.translate',
+  ])
+  .component('ovhManagerProductOffers', component)
+  .constant('OVH_MANAGER_PRODUCT_OFFERS_PRICING_CONSTANTS', pricingConstants)
+  .constant('OVH_MANAGER_PRODUCT_OFFERS_WORKFLOW_CONSTANTS', workflowConstants)
```

Comment:
I'm wondering if there is the need for those constants outside the component

Reply:
Yes it is to avoid duplicating constants in the scope where the component is used